README.md

```diff
@@ -1,74 +1,270 @@
 # Kinvey CLI
-> Utility for deploying and managing FlexServices on the Kinvey FlexService Runtime
+
+Kinvey CLI is an utility for deploying and managing FlexServices running on the Kinvey FlexService Runtime.
 
 ## Installation
-`npm install -g kinvey-cli`
+
+Kinvey CLI is distributed as an NPM package. After you install NPM, run the following command to download and install Kinvey CLI.
+
+    npm install -g kinvey-cli
 
 ## Usage
-In your project directory, run `kinvey config` to set up your project. The CLI will prompt for Kinvey credentials, app or organization, and an Internal Flex Service to deploy to.
-
-### Commands
-* `config [instance]` - set project options (including optional dedicated Kinvey instance name).
-* `deploy` - deploy node project in the current directory as an Internal Flex Service. You can check the status of a deploy using the `job` command (more info below).
-* `status` - display the health and deployment metadata (version, deployer info, requested at) of a deployed Internal Flex Service.
-  * E.g. `ONLINE [v0.0.48]`
-* `help` - display usage information.
-  * To display help/usage info for a particular command, use `kinvey <command> -h`
-* `list` - list Internal Flex Services for the current app.
-* `logs` - query logs for the current Internal Flex Service
-  * Run `kinvey logs -h` for advanced `logs` command usage details, including returning more than the default number of entries, specifying a page number, or filtering by timestamp.
-    * **Note:** Logs calls return 100 entries by default and can return up to 2,000 entries.
-  * Logs are displayed in the following format: `<runtime id> <timestamp> - <message>`
-    * E.g. `ac7df839104d 2016-02-23T20:00:29.334Z - hello world`
-* `logout` - clears Kinvey session data and project settings.
-* `job [id]` - return the status of a `deploy` or `recycle` command. Returns the status of the most recent `deploy` or `recycle` command if `[id]` is not specified.
-
-### Options
-* `-e, --email <e-mail>` - e-mail address of your Kinvey account.
-* `--host <host>` - set host of the Kinvey service.
-* `-p, --password <password>` - password of your Kinvey account.
-* `-s, --silent` - do not output anything.
-* `-c, --suppress-version-check` - do not check for package updates.
-* `-v, --verbose` - output debug messages.
-
-### Credentials
-
-Kinvey credentials can be provided as command line options or as environment variables.
-
-#### Command line options
-Run `kinvey config` and the CLI will prompt for Kinvey credentials if they are not already set as environment variables.
-
-You can also create a one-time session using the `-e` and `-p` (and optionally `--host`) flags. Set these parameters explicitly in your calls if you are unable to init with `kinvey config [host]`. It's important to note that the `-e` and `-p` flags take precedence over environment variables. Example:
+
+    kinvey `<command>` [args] [options]
```

Comment:
I think backticks should start before 'kinvey' and end after '[options]'
https://github.com/Kinvey/kinvey-cli/blob/f6fc2cde184d0ad1a4a61ba0762256114cab36fa/README.md

Reply:
👍